### DISCUSSION TITLE
Possibility to add a native android password hide/show button on TextFields

### DISCUSSION THREAD

**Discussion by jordanbisato** [appcelerator/titanium_mobile#13756] [General]
Hi,

I was searching about hide and show button for password masked TextField and i found these links:

https://stackoverflow.com/questions/44173249/change-to-custom-icon-from-eye-icondefault-for-hide-show-password-in-android-e

https://stackoverflow.com/a/39022642/4216870

I looked at TiUIText file, and we already use TextInputLayout in some cases. 
(https://github.com/tidev/titanium_mobile/blob/918388a75a98c3f5458d27f6ed035f11883b303e/android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java)

So, its possible to implement this solution, or another solution to Titanium SDK?

**Comment by m1ga** [comment]
there was a PR https://github.com/tidev/titanium_mobile/pull/12712 but it was closed in favor of a JS implementation that will work for both platforms.
